eng/tools/compare-packages.ps1

```diff
@@ -7,36 +7,55 @@ param (
 
 function ExtractTGZPackages($pathToPkg){
   $regExp = "-\d+\.\d+\.\d+(-\w*\.\d*)?(-dev-$dailyDevBuildNo)?"
+
+
+
+  ## $parentCurrent = Split-Path -Path $pathToPkg -Parent
+  ## $parentExtractName = (Split-Path -Path $pathToPkg -Leaf)+"-contents"  
+  ## $parentExtractDir = Join-Path $parentCurrent -ChildPath $parentExtractName
+
+
+  $parentExtractDir = Join-Path $pathToPkg -ChildPath "all-contents"
+  mkdir "$parentExtractDir"
   foreach ($p in $(dir $pathToPkg -r -i *.tgz)){
-    $extractDir = (get-item $p.FullName).Directoryname + "/" + $p.BaseName
+    $extractDir = Join-Path $parentExtractDir -ChildPath $p.BaseName
```

Comment:
FYI you don't need to specify the `-ChildPath` argument, it is redundant.

Reply:
removed